website/src/content/docs/zh-tw/getting-started/installation.mdx

```
@@ -0,0 +1,169 @@
+scoop install superfile
+```
+
+## 社群維護的套件
+
+[![Packaging status](https://repology.org/badge/vertical-allrepos/superfile.svg)](https://repology.org/project/superfile/versions)
+
+> 依字母排序
+
+### Arch
+
+###### 從原始碼建置套件
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h6

(MD001, heading-increment)